signalfx-yammer-example/src/main/java/com/signalfx/yammer/example/App.java

```diff
@@ -112,13 +115,29 @@ public Double value() {
 				.withDimension("Test-Dimension", "_Gauge")
 				.withMetricType(SignalFxProtocolBuffers.MetricType.GAUGE);
 		
+		// Timer
+		
+		MetricName timerName = new MetricName("group1", "type1", "timer");
+		Timer timer = metricRegistery.newTimer(timerName, TimeUnit.SECONDS, TimeUnit.SECONDS);
+		
+		metricMetadata.forMetric(timer)
+				.withMetricName("Test-Metric") // To see metric for the timer you should look for
+											   // Test-Metric.max / Test-Metric.min
```

Comment:
This comment didn't render very well. Can you re-format? Perhaps up where the // Timer is and do a /* *** */ block.  Thanks.